src/main/java/org/jitsi/impl/protocol/xmpp/colibri/ColibriConferenceImpl.java

```diff
@@ -445,17 +452,73 @@ protected Stanza sendAllocRequest(String endpointId,
                                       ColibriConferenceIQ request)
         throws ColibriException
     {
-        try
+        long start = System.nanoTime();
+        Stanza reply = sendAndGetReply(request);
+        long end = System.nanoTime();
+        stats.allocateChannelsRequestTook(end - start);
+        return reply;
+    }
+
+    /**
+     * Send the given {@link IQ} and return the reply, using the JVB API
+     * if the API is configured, otherwise the XMPP connection
+     * @param iq the IQ to send
+     * @return the reply
+     * @throws ColibriException in case of error
+     */
+    protected Stanza sendAndGetReply(IQ iq)
+        throws ColibriException
+    {
+        Stanza reply;
+        if (jvbApi != null)
+        {
+            if (logger.isDebugEnabled())
+            {
+                logger.debug("Using JVB API for request");
+            }
```

Comment:
I don't think you'll want this log on every request - is there a way to log it once?

Reply:
Well, it is on debug :) But I could log it in `setJvbApi` instead?

Reply:
I think it'd make sense - if you have debug logs on because you're debugging something, it'll get very noisy.